src/styles/blocks/watch.scss

```diff
@@ -0,0 +1,70 @@
+.stopwatch {
+  width: 80vmin;
+  height: 80vmin;
+  position: relative;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  border: $borderWatch;
+  border-radius: 50%;
+
+  &__center {
+    width: 5vmin;
+    height: 5vmin;
+    background-color: $colorCenter;
+    border-radius: 50%;
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+    z-index: 2;
+  }
+
+  &__hand {
+    position: absolute;
+    bottom: 50%;
+    left: 50%;
+    transform-origin: bottom left;
+    transform: rotate(0deg) translateX(-50%);
+
+    &--seconds {
+      width: 1.5vmin;
+      height: 38vmin;
+      background-color: $colorSecond;
+      animation: rotate-seconds 60s linear infinite;
+    }
+
+    &--minutes {
+      width: 3vmin;
+      height: 20vmin;
+      background-color: $colorMinute;
```

Comment:
The variable `$colorMinute` should be written in kebab-case as `$color-minute` to follow the [Code Style Rules](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules).